doc/distrib/NodeHelpFiles/_review/Revit.Elements.FamilyInstance.Location.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`FamilyInstance.Location` returns a Dynamo point for the given family instance. If there is no location, a null value is returned.
```

Comment:
I think we should  point out here that this node works when Family instance has location that is a point location, i.e., a LocationPoint class object like A footing, a door, or furniture. And it will return a null if the location of the family instance is a LocationCurve class object like a beam, or a wall. I think the example is eluding to this, but lets make it explicit.